waspc/src/Generator/Job/IO.hs

```diff
@@ -19,6 +19,7 @@ printJobMessage jobMsg = do
     let prefix = case J._jobType jobMsg of
             J.Server -> "\ESC[35mServer:\ESC[0m " -- Magenta
             J.WebApp -> "\ESC[36mWeb app:\ESC[0m " -- Cyan
+            J.Db     -> "\ESC[32mDb:\ESC[0m " -- Green
```

Comment:
I was avoiding green and red because they are indicative of success or failure. Maybe use some other color? Magenta, cyan and yellow are taken. Maybe gray :D?

Reply:
There is no gray within basic 8 colors that are supported by most terminals so I put white. Although in my terminal it looks kinda blueish, not sure way. All colors look slightly different to me, maybe because of some theme that is applied on the terminal, I am not sure?